File: src/handlers/index/create.py
import json
# import src.openai_embeddings.request
import os
def handler(event, context):

    body = event['body']

    key = os.environ.get('EMBDEDDINGS_OPENAI_API_KEY')
    data = "This is a large string..."
    # src.openai_embeddings.request.make_http_call_with_chunking(data, "text-embedding-ada-002", key)
    response = {
        "statusCode": 200,
        "body": json.dumps(data)
    }

    return response

File: src/handlers/index/create.js
const {makeHttpCallWithChunkings} = require("../../functions/make-http-call-with-chunking");
module.exports = {
    handler:async (event) =>{
        try {
            console.log(process.env.EMBDEDDINGS_OPENAI_API_KEY)
            const result = await makeHttpCallWithChunkings(event.body,process.env.EMBDEDDINGS_OPENAI_API_KEY)
            console.log(result)
            return {
                statusCode:200

            }
        } catch (e){
            console.log(e)
            return {
                statusCode:500

            }
        }

    }
}
